tools/arcilator/arcilator.cpp

```diff
@@ -528,11 +598,26 @@ int main(int argc, char **argv) {
   registerPassManagerCLOptions();
   registerDefaultTimingManagerCLOptions();
   registerAsmPrinterCLOptions();
-  cl::AddExtraVersionPrinter(
+  llvm::cl::AddExtraVersionPrinter(
       [](raw_ostream &os) { os << getCirctVersion() << '\n'; });
 
   // Parse pass names in main to ensure static initialization completed.
-  cl::ParseCommandLineOptions(argc, argv, "MLIR-based circuit simulator\n");
+  llvm::cl::ParseCommandLineOptions(argc, argv,
+                                    "MLIR-based circuit simulator\n");
+
+  if (!runSimulation.empty()) {
```

Comment:
Alternately don't even have the cmdline option if jit is not enabled.

Reply:
I am worried that if the command line option disappears, it may create confusion for users that built CIRCT without knowing about the conditional build, and believe that the JIT interface has changed or does not exist.